### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by Juliocho990**
### Summary
- CSS grid was implemented
- Book detail view was refactored to achieve a responsive card with the 1024px breakpoint

### Screenshot
grid refactoring

### Trello card
https://trello.com/c/eVgsfbZV/13-dise%C3%B1o-responsive

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -1,57 +1,139 @@
...
+  font-weight: bold;
+  grid-area: 2 / 1 / 3 / 3;
+  justify-self: center;
+  text-align: center;
+
+  @include for-mobile {
+    border-bottom: 4px solid $las-palmas;
+    font-size: 16px;
+    grid-area: 3 / 1 / 4 / 3;
+    line-height: 2.2;
```
Comment (CrisVega08) [inline scss/detail.scss:70]: use 2 or 3

File: scss/detail.scss [hunk 2]
```diff
@@ -1,57 +1,139 @@
...
+.grid__card {
   background-color: $white;
   box-shadow: 4px 4px 10px 0 rgba($black, 0.5);
-  display: flex;
-  max-width: 900px;
-  padding: 24px;
-  width: 100%;
-
+  display: grid;
+  grid-template: 40px 40px 1fr / 35% 1fr;
```
Comment (CrisVega08) [inline scss/detail.scss]: I dont like that combine px fr and percentage, try to change percentage value

File: scss/detail.scss [hunk 3]
```diff
@@ -1,57 +1,139 @@
...
+  padding: 32px;
+  
   @include for-mobile {
-    flex-direction: column;
-    max-width: 336px;
+    grid-template: 40px 1fr 40px 120px/ 35% 1fr;
```
Comment (CrisVega08) [inline scss/detail.scss]: same

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: scss/detail.scss [hunk 4]
```diff
@@ -1,57 +1,139 @@
...
-  align-items: center;
-  display: flex;
-  flex-direction: column;
-  flex-grow: 1;
-  justify-content: center;
+.grid__wrapper {
+  display: grid;
+  grid-template: 1fr / 1fr;
+  height: 100%;
+  position: absolute;
```
Comment (eledwinn) [inline scss/detail.scss]: Why have absolute positioning? This element is a body child, can you try without absolute?

**Review by eledwinn — CHANGES_REQUESTED** [review]

File: scss/detail.scss (hunk 2 above)
Comment (Juliocho990) [reply scss/detail.scss]: ok

File: scss/detail.scss (hunk 3 above)
Comment (Juliocho990) [reply scss/detail.scss]: ok

File: scss/detail.scss (hunk 4 above)
Comment (Juliocho990) [reply scss/detail.scss]: I tried to remove la position property, but I did not get the result I wanted. While, when I use **position: absolute** property I can fill the full height of the body.

**Review by eledwinn — APPROVED** [review]

**Review by CrisVega08 — APPROVED** [review]

**Pull request merged**